### ISSUE TITLE
check_mssql_database.py - type 'exceptions.ZeroDivisionError'

### ISSUE THREAD

**Issue by box293** [NagiosEnterprises/check_mssql_collection#15]
```
./check_mssql_database.py -H mssql01.box293.local -U 'nagios' -P 'xxxxxx' -T 'Sample' -p 14333 --logcachehit --warning 0:95 --critical 0:97
<type 'exceptions.ZeroDivisionError'>
Caught unexpected error. This could be caused by your sysperfinfo not containing the proper entries for this query, and you may delete this service check.
```

Tested against an SQL 2016 server.

Perhaps a better error message or don't try and divide by zero.

**Comment by jomann09** [comment]
It's the issue with trying to grab data the way it is on SQL 2016 servers. We really need to figure out a good way of knowing what version the server is so we can appropriately query.

**Comment by jomann09** [comment]
This is fixed in 2.1.1 of the check_mssql_database.py script.

**Comment by box293** [comment]
Still an issue for me:
```
[root@xi-c7x-x64 tmp]# cd /var/tmp/

[root@xi-c7x-x64 tmp]# wget https://raw.githubusercontent.com/NagiosEnterprises/check_mssql_collection/master/check_mssql_database.py
--2017-08-10 07:29:21--  https://raw.githubusercontent.com/NagiosEnterprises/check_mssql_collection/master/check_mssql_database.py
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.28.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.28.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 14831 (14K) [text/plain]
Saving to: ‘check_mssql_database.py’

100%[===========================================================================>] 14,831      --.-K/s   in 0.04s   

2017-08-10 07:29:21 (378 KB/s) - ‘check_mssql_database.py’ saved [14831/14831]

[root@xi-c7x-x64 tmp]# chmod +x check_mssql_database.py

[root@xi-c7x-x64 tmp]# grep Version check_mssql_database.py # Version    : 2.1.1

[root@xi-c7x-x64 tmp]# ./check_mssql_database.py -H mssql01.box293.local -U 'nagios' -P 'xxxxx' -T 'Sample' -p 14333 --logcachehit --warning 0:95 --critical 0:97
<type 'exceptions.ZeroDivisionError'>
Caught unexpected error. This could be caused by your sysperfinfo not containing the proper entries for this query, and you may delete this service check.

[root@xi-c7x-x64 tmp]# echo $?
3

```

**Comment by jomann09** [comment]
As previously stated this seems to be an MSSQL 2016 issue with the plugin, the error fixed does not address this particular incompatibility.

**Comment by jomann09** [comment]
Fixed this here 9a29a0e5403ac5e3fb2565ee2f63b7f701fb904c
